src/components/Sidebar/Sidebar.js

```diff
@@ -22,7 +22,7 @@ const Sidebar = () => {
         { children.map((child, i) =>
           !!child.phase1 && (
             <Link to={ child.linkTo } key={ i }>
-              <div className="sidebar__link">{ child.name }</div>
+              <div className={`sidebar__link ${(pathname === child.linkTo)?'sidebar__link--active':''}`}>{ child.name }</div>
```

Comment:
i dont love doing a ternary and returning `''`, could we do something like 
```
<div className={'sidebar__link'} data-active={!!(pathname === child.linkTo)} >{ child.name }</div>
```

Reply:
this will create `data-active='true'` or data-active='false'` and we can target that it in scss like:
```
&:hover, &[data-active='true'] {
    background-color: $purple;
    color: $white;
  }
  ```

Reply:
You're so next level Mateo.